### PR TITLE
Add Left Right Views for input

### PR DESCRIPTION
**Pull request by ElishaSamPeterPrabhu**
Screenshot_1697710457

### CODE REVIEW

File: Trimble.Modus.Components/Controls/TMInput/TMInput.xaml [hunk 1]
```diff
@@ -74,27 +60,14 @@
...
-      </ImageButton.IsEnabled>
-    </ImageButton>
+    </controls:BorderlessEntry>
+    <ContentView x:Name="rightCustomView"
+                 VerticalOptions="Center"
+                 HorizontalOptions="Center"
+                 Grid.Row="0"
+                 Grid.Column="2"
+                 HeightRequest="40"
+                 Content="{Binding RightCustomContent}">
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/TMInput/TMInput.xaml:70]: Still says `RightCustomContent `?

File: Trimble.Modus.Components/Controls/TMInput/TMInput.xaml.cs [hunk 2]
```diff
@@ -162,8 +188,8 @@ private void InputBorderlessEntry_Unfocused(object sender, FocusEventArgs e)
     /// </summary>
     internal void ToggleRightIconState(bool enabledState)
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/TMInput/TMInput.xaml.cs:189]: Do we need these methods at all?

**Review by rthanga1 — CHANGES_REQUESTED** [review]

File: Trimble.Modus.Components/Controls/TMInput/TMInput.xaml.cs (hunk 2 above)
Comment (ElishaSamPeterPrabhu) [reply Trimble.Modus.Components/Controls/TMInput/TMInput.xaml.cs:189]: Yes, a bit modified for number input

File: Trimble.Modus.Components/Controls/TMInput/TMInput.xaml (hunk 1 above)
Comment (ElishaSamPeterPrabhu) [reply Trimble.Modus.Components/Controls/TMInput/TMInput.xaml:70]: Changed to RightView and left view

**Comment by ElishaSamPeterPrabhu** [comment]
@rthanga1 For the left view and right view the name for the commands are still left icon and right icon commands as usually icons only serve purpose for a command , or should we change it?

**Comment by rthanga1** [comment]
> @rthanga1 For the left view and right view the name for the commands are still left icon and right icon commands as usually icons only serve purpose for a command , or should we change it?

@ElishaSamPeterPrabhu We don't even need those commands.

**Comment by ElishaSamPeterPrabhu** [comment]
> > @rthanga1 For the left view and right view the name for the commands are still left icon and right icon commands as usually icons only serve purpose for a command , or should we change it?
> 
> @ElishaSamPeterPrabhu We don't even need those commands. 

@rthanga1 I added those commands since , number input inherits the inner borderless entry , left and right icons form TMInput .

**Comment by rthanga1** [comment]
I am still thinking we should somehow support just the iconsource along with the left/right views. I will checkout this tonight and update you if anything can be improved. Otherwise I will approve this PR.

**Review by rthanga1 — APPROVED** [review]

**Comment by ElishaSamPeterPrabhu** [comment]
Modify password icon to a functional one
Screenshot_1698210455
Screenshot_1698210453

**Review by rthanga1 — APPROVED** [review]

**Pull request merged**